python/cudf/cudf/core/column_accessor.py

```diff
@@ -537,6 +537,27 @@ def rename_column(x):
 
         return self.__class__(ca)
 
+    def droplevel(self, level):
```

Comment:
Note that `ColumnAccessor` _is_ our implementation of "multi-level columns". The `.columns` attribute of a cudf.DataFrame doesn't return a `cudf.MultiIndex`, but rather a `pd.MultiIndex`. So operations like dropping a level of a multi-level column need to be either methods of `ColumnAccessor` or free functions that act on `ColumnAccessor`.

Reply:
Ah of course. Yeah I guess we're stuck with this. I dislike the methods in ColumnAccessor that are used very sparingly since I'd like to thin out this class rather than augment it, but for now I think we live with the result.